src/components/Banner/BaseAnnouncementBanner/styled.tsx

```diff
@@ -0,0 +1,84 @@
+import walletBannerPhoneImageSrc from 'assets/images/wallet_banner_phone_image.png'
+import { BaseButton } from 'components/Button'
+import { OpacityHoverState } from 'components/Common'
+import Row from 'components/Row'
+import { X } from 'react-feather'
+import styled from 'styled-components'
+import { Z_INDEX } from 'theme/zIndex'
+
+export const PopupContainer = styled.div<{ show: boolean }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  ${({ show }) => !show && 'display: none'};
+
+  background: url(${walletBannerPhoneImageSrc});
+  background-repeat: no-repeat;
+  background-position: top 18px right 15px;
+  background-size: 166px;
+
+  :hover {
+    background-size: 170px;
+  }
+  transition: background-size ${({ theme }) => theme.transition.duration.medium}
+    ${({ theme }) => theme.transition.timing.inOut};
+
+  background-color: ${({ theme }) => theme.chain_84531};
+  color: ${({ theme }) => theme.textPrimary};
+  position: fixed;
+  z-index: ${Z_INDEX.sticky};
+
+  padding: 24px 16px 16px;
+
+  border-radius: 20px;
+  bottom: 20px;
+  right: 20px;
+  width: 390px;
+  height: 164px;
+
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+
+  box-shadow: ${({ theme }) => theme.deepShadow};
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
+    bottom: 62px;
+  }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
+    background-position: top 32px right -10px;
+    width: unset;
+    right: 10px;
+    left: 10px;
+    height: 144px;
```

Comment:
this wasn't in the UniswapWalletBanner `PopupContainer` style, making sure this is intentional?

Reply:
yeah, this is a new addition to this component